### ISSUE TITLE
2.6.9 Patch Release Plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2623]
- #2609
- #2435 
- #2605 
- #2618 

## Pick list
```
git cherry-pick 4fced3536779b4cbae5c9e9c90776b31c2ae450e
git cherry-pick 56d68a2cde06622e7748291485a6c4a81e731153
git cherry-pick 57e73d9177f751cb0b4bf1c054b5e267a2095b4b
```

**Comment by joeparsons** [comment]
## Release notes snippet
`git log 2.6.8..HEAD --pretty='format:- %s'`
```
- Closes #2617 Remove the back slash from the analytics dashboard link (#2618)
- Closes #2604 Change port number in az_mail module from 587 to 2587 (#2605)
- Closes #2433 Add users to ManualMigrationLookup (#2435)
- Closes #2608 Update drupal/core-recommended to 9.5.10 (2.6.x only) (#2609)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.6.9
